src/PMGDQueryHandler.cc

```diff
@@ -1021,6 +1021,38 @@ void insert_into_queue(std::list<AutoDeleteNode *> *queue,
   }
 }
 
+void PMGDQueryHandler::print_node_idx_stats(char *tag_name, char *prop_id) {
```

Comment:
Do we use this function somewhere or is this just a nice-to-have function to get information regarding node?

Reply:
Not yet used, but handy for debug purposes.

Reply:
Yes we need to test it for coverage in case people use it

Reply:
You got it!